oneflow/core/functional/functional_api.yaml

```
@@ -1259,7 +1259,11 @@
   bind_python: true
 
 - name: "masked_fill"
-  signature: "Tensor (Tensor x, Tensor mask, Scalar value) => MaskedFill"
+  signature: "Tensor (Tensor input, Tensor mask, Scalar value) => MaskedFill"
```

```suggestion
  signature: "Tensor (Tensor input, Tensor mask, Scalar value) => MaskedFill"
```

这个直接导出给用户的接口，形参名没对齐，顺便改了。